package.json

```diff
@@ -37,7 +37,7 @@
     "lodash": "^4.17.21",
     "madr": "2.1.2",
     "markdown-it": "10.0.0",
-    "markdown-it-katex": "github:MicroDroid/markdown-it-katex",
```

Comment:
in general, updating our dependencies is a good idea :) for this specific dependency, a fork of the katex library was used since it fixes some issues related to specific math symbols
for example, $\underbrace{x}$ is rendered correctly with the github:MicroDroid/markdown-it-katex (see master) but not in the ^2.0.3 version (just use the snippet I mentioned as a test case)

since we use a lot of math symbols in quantum computing patterns we decided to use  the working version for now - could you roll back the version and add a comment that explains why we do that?

Reply:
The reason why I have changed it to ^2.0.3 is because of the lint error.  In the phase where Node.js dependencies are installed npm was throwing an error that it could not access `github:MicroDroid/markdown-it-katex`. I guess this fork is private now. I am assuming that our master will get the same issue if we rerun lint.